server/node-server.js

```diff
@@ -51,8 +45,10 @@ app.get('/auth/facebook/callback',
     session: false,
   }),
   (request, response) => {
-    response.redirect('/auth/facebook?access_token=' + request.user.token
-     + '&display_name=' + request.user.displayName);
+    const { token, displayName } = request.user;
+    response.redirect(
+      `/login/facebook?access_token=${token}&display_name=${displayName}`
```

Comment:
Changed the front-end route to `/login/facebook`. I was getting weird errors because we have a server-side `/auth/facebook` route as well.